src/engraving/dom/tie.h

```diff
@@ -25,6 +25,64 @@
 #include "slurtie.h"
 
 namespace mu::engraving {
+class TieEndPointList;
+class TieEndPoint
```

Comment:
Side question: would TieJumpPoint be perhaps a better name for this class? I know what you mean by TieEndPoint, but it feels to me like it doesn't communicate the main purpose. Also because we often call "end point" the graphical end points too.